packages/babel-core/src/util.js

```diff
@@ -80,8 +78,8 @@ export function regexify(val: any): RegExp {
 
 export function arrayify(val: any, mapFn?: Function): Array<any> {
   if (!val) return [];
-  if (isBoolean(val)) return arrayify([val], mapFn);
-  if (isString(val)) return arrayify(list(val), mapFn);
+  if (typeof val === "boolean") return arrayify([val], mapFn);
+  if (typeof val === "string") return arrayify(list(val), mapFn);
 
```

Comment:
It's easier to stick with `typeof` than helpers, so you end up with consistent code, and less nits on PRs.